src/hamiltonian.jl

```diff
@@ -167,18 +167,6 @@ end
 
 energy(args...) = -neg_energy(args...)
 
-####
-#### Momentum refreshment
-####
-
-function phasepoint(
```

Comment:
On second thought, let's revert the change to the `phasepoint` function, since the current function is meant to support vectorised HMC. In comparison, the proposed changes in this PR are a minor regression.

Reply:
Oh yeah, that would lose the vectorized HMC support, just reverted the change.